cpp/tensorrt_llm/kernels/cutlass_kernels/include/common.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace tensorrt_llm::kernels::cutlass_kernels
```

Comment:
Lets change this namespace to be a public one so we don't need to do the using namespace hack everywhere we use it.
There is no reason for this to be different in internal cutlass and public since this is part of the API

Reply:
Yes I agree, my point is this doesn't need to be duplicated internally and public, we can just have a public version and put it in the regular common/ directory and reuse it public & private